cs3/storage/provider/v1beta1/resources.proto

```diff
@@ -135,29 +141,28 @@ enum LockType {
 // of this metadata according to their constraints, a reference
 // implementation is given here. The lock SHOULD be stored
 // as an extended attribute on the referenced filesystem entry.
-// It MUST NOT be accessible via the Stat/SetArbitraryMetadata APIs,
-// and it SHOULD contain a base64-encoded JSON with the following format:
-// {
-//   "type" : "<LOCK_TYPE>",
-//   "h" : "<holder>",
-//   "md" : "<metadata>",
-//   "mtime" : "<Unix timestamp>"
-// }
+// It MUST NOT be accessible via the Stat/SetArbitraryMetadata APIs.
```

Comment:
So the `ResourceInfo` returned by stat calls would always have `lock` and `advisory_locks` empty?

Reply:
Yes for `advisory_locks` as we're not foreseeing to implement them for now. The `lock` may be filled if the file is actually locked (by WOPI).

Reply:
Then this comment should be removed?

Reply:
Hmm correct, the comment referred to the suggested xattr-based implementation. Let me better clarify it.